Makefile

```diff
@@ -114,6 +115,14 @@ $(PROGRAM): $(GO_FILES)
 	@echo "Building: $(PROGRAM), version: $(SVERSION)..."
 	time go build -i -ldflags "-X main.program=$(PROGRAM) -X main.version=$(SVERSION)" -o $(PROGRAM) $(BUILD_FLAGS);
 
+define buildrelease
```

Comment:
wow. I am always impressed when someone uses new `make` magic that I don't yet know!

Thank you wizard!

Reply:
Well, I cannot take full credit, it is based upon some other cross build functions I mixed up together.